_docs/_latest/use/cli/enforcer.md

```diff
@@ -17,40 +17,41 @@ Additional enforcement endpoints are in development.
 
 ## Before you begin
 
-Forseti Enforcer requires write permissions for the resources that it manages.
-When you setup Forseti using the installer, a service account is created and
+Enforcer requires write permissions for the resources that it manages.
+When you set up Forseti using the installer, a service account is created and
 granted write access to update firewalls on any project in your organization.
 
-If you did not use the installer, you did not enable enforcer during
-installation, or you want to run enforcer on a different instance than the
-Forseti server, then you need to grant the service account used by enforcer
-the following permissions:
+If you didn't use the installer, or you didn't enable Enforcer during
+installation, or if you want to run Enforcer on a different instance than the
+Forseti server, then you need to grant following permissions to the service
+account used by Enforcer:
 
 * The Compute Security Admin role on all projects that it will enforce, or on
-  the folder(s) or organization parenting the projects.
+  the folders or organization parenting the projects.
 * The cloud-platform or compute API scope on the instance that's running
   Forseti Enforcer.
 
-## Using Forseti Enforcer
+## Using Enforcer
```

Comment:
Same comment as above, let's look for links to this heading.

Reply:
Possible to redirect or use persistent anchor here?

Reply:
Resolved.